docs/execution/execution.md

```diff
@@ -11,6 +11,15 @@ If ScubaGear was installed by [downloading from GitHub](../installation/github.m
 Import-Module -Name .\PowerShell\ScubaGear 
 ```
 
+## Install Dependencies
+
+ScubaGear requires a number of dependencies to be installed before it can be invoked.  To install the dependencies:
```

Comment:
If you install ScubaGear via PSGallery, is this step still necessary? (I'm actually not sure what the answer is, but if it's not it would be good to say that). Similar to how you only need to run `Import-Module` if you didn't install via PSGallery.

Reply:
Yes, it does.  In fact, that's the entire point of this PR:  Making it explicitly clear that you need to do this.